extension/intents/music/music.toml

```diff
@@ -143,3 +143,12 @@ phrase = "play music"
 [[music.unpause.example]]
 phrase = "play spotify"
 test = true
+
+[music.find]
+ description = "Find the tab that's currently making sound"
+match = """
+  (what | what's | which ) ( tab | song | music | title ) ( is ) playing 
```

Comment:
`( is )` means the same as `is`. I think it's supposed to be an optional word though, so you need to use `|` to say that no-word is an option, like `(is | )`

Also I think tab/song/music/title could be optional too.